packages/core/src/messages/responses/cache-dictionary-fetch.ts

```diff
@@ -56,17 +56,6 @@ class _Hit extends Response {
     return this.valueMapStringString();
   }
 
-  /**
-   * Returns the data as a Map whose keys and values are byte arrays.
-   * @returns {Map<Uint8Array, Uint8Array>}
-   */
-  public valueMapUint8ArrayUint8Array(): Map<Uint8Array, Uint8Array> {
-    return this.items.reduce((acc, item) => {
-      acc.set(item.field, item.value);
-      return acc;
-    }, new Map<Uint8Array, Uint8Array>());
-  }
```

Comment:
Was this intended in this PR?

Reply:
yes, sorry, I left it on the commit message, but now realizing that that didn't show up very visibly in the PR itself. I updated the PR description to include my notes from the commit message.